### PR TITLE
rocAL - Python Unit Test Fix PR

### PR DESCRIPTION
**Pull request by swetha097**
This PR fixes issues with 
- Tensorflow classification and detection in the GPU backend
- ResizeMirrorNormalize and Vignette augmentation outputs

### CODE REVIEW

File: rocAL/rocAL_pybind/examples/READERS_TEST_FILE.sh [hunk 1]
```diff
@@ -168,7 +168,7 @@ if [[ rocAL_api_caffe_reader -eq 1 ]]; then
 
     # Mention dataset_path
     # Classification
-    data_dir=$ROCAL_DATA_PATH/caffe/classification/ilsvrc12_train_lmdb/
+    data_dir=$ROCAL_DATA_PATH/caffe/classification/
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/examples/READERS_TEST_FILE.sh:171]: @swetha097 Have these paths been updated in the mivisionx-data repo as well?

**Review by LakshmiKumar23 — COMMENTED** [review]
@swetha097 Check codacy warnings

File: rocAL/rocAL_pybind/examples/READERS_TEST_FILE.sh (hunk 1 above)
Comment (swetha097) [reply rocAL/rocAL_pybind/examples/READERS_TEST_FILE.sh:171]: Yes Lakshmi, that has been updated

**Comment by LakshmiKumar23** [comment]
@swetha097 as always, I cannot get the gpu backend running for unit tests. It's the same sh vs bash vs ./PYTHON... issue

**Comment by kiritigowda** [comment]
@LakshmiKumar23 let me know if this can make it in today's merge?

**Comment by LakshmiKumar23** [comment]
@kiritigowda Can be merged. tested TF classification and detection on rocal-gpu backend

**Review by LakshmiKumar23 — APPROVED** [review]

**Pull request merged**